.github/workflows/beekeeper.yml

```diff
@@ -14,7 +14,7 @@ env:
   SETUP_CONTRACT_IMAGE: "ethersphere/bee-localchain"
   SETUP_CONTRACT_IMAGE_TAG: "0.9.1-rc3"
   BEELOCAL_BRANCH: "main"
-  BEEKEEPER_BRANCH: "master"
+  BEEKEEPER_BRANCH: "fix/staked-amount-rename"
```

Comment:
since the beekeeper PR has been merged, you can do a new beekeeper release and put this back to master

Reply:
It is in the works, just to have a new tag release on beekper